doc/nimgrep.rst

```diff
@@ -2,17 +2,20 @@
   nimgrep User's manual
 =========================
 
+.. default-role:: literal
```

Comment:
why not code? it renders differently for things like backslash IIRC; code is the closest to verbatim rendering (and to markdown)

`.. default-role:: code` is what I've used in https://github.com/nim-lang/Nim/pull/17028 (and manual since https://github.com/nim-lang/Nim/pull/17259)

Reply:
Because code is intended for code in programming languages. It expects highlighting. This is an actually implemented feature in rst2html.py:

```rst
.. role:: nim(code)
   :language: nim

.. default-role:: nim

Some code: `when PrintRopeCacheStats: echo "rope cache stats: "`.

Another example: `from ic / ic import rodViewer`.

```

![image](https://user-images.githubusercontent.com/1299583/111888326-8f09bf80-89ec-11eb-933e-a8d9f7068436.png)

It uses pygmentize to highlight code. And pygmentize knows Nim!

I think one day we will implement that too.

...And here in `nimgrep.rst` all quotes are used either for option names or some commands, which are not exactly code (may be viewed as Bash code though or any other shell without specifics). All those cases match `literal` role perfectly.

Reply:
ok! maybe keep this comment open as it has useful information (or link to it from https://github.com/nim-lang/Nim/issues/17340)